src/deployconfig.jl

```diff
@@ -104,6 +104,18 @@ marker(x) = x ? "✔" : "✘"
 
 env_nonempty(key) = !isempty(get(ENV, key, ""))
 
+
+"""
+    CommonDevBranches
+
+A stand-in for the name of the devbranch. Rather than hardcoding to `master` or `main` this
+accepts any of `master`, `main`.
+"""
+struct CommonDevBranches end
+
+matches_devbranch(branchname, devbranch) = branchname == devbranch
```

Comment:
I considered implementing it via overloading `==(x::AbstractString, CommonDevBanches)= x ∈ ("main", "master")`
but I decided that that was too magic.
Also it would break `Dict` since `hash` wouldn't work right.
Not that it seems likely this would end up in a `Dict` but no sense being weird if it does.